src/rendering/renderer.js

```
@@ -0,0 +1,157 @@
+  }
+}
+/**
+ * Fills the canvas with the configured background color and optionally draws a grid overlay aligned with the camera position.
+ */
+export function drawBackground() {
+  // Fill background with light green
+  ctx.fillStyle = config.colors.background;
+  ctx.fillRect(0, 0, canvas.width, canvas.height);
+
+  // Draw grid if enabled
+  if (config.colors.grid && config.colors.grid.enabled) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)